AgendaUWP/AgendaUWP/ViewModels/DetailContactPageViewModel.cs

```diff
@@ -0,0 +1,75 @@
+﻿using AgendaUWP.Models;
+using Prism.Commands;
+using Prism.Windows.Mvvm;
+using Prism.Windows.Navigation;
+using Service;
+using Service.Model;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AgendaUWP.ViewModels
+{
+    public class DetailContactPageViewModel : ViewModelBase
```

Comment:
Could you please fix indentation and blank lines?

Reply:
Ok! Done!